packages/mobile/package.json

```diff
@@ -12,12 +12,18 @@
     "lint": "eslint --ext .jsx,.js,.ts,.tsx ./src/ --fix",
     "lint-ci": "eslint --ext .jsx,.js,.ts,.tsx ./src/",
     "gen": "plop",
-    "prepare-backend-assets": "mkdir -p ./nodejs-assets/nodejs-project && cp ../backend/lib/bundle.cjs ./nodejs-assets/nodejs-project/bundle.cjs",
+    "prepare-backend-assets": "run-script-os",
+    "prepare-backend-assets:default": "mkdir -p ./nodejs-assets/nodejs-project && cp ../backend/lib/bundle.cjs ./nodejs-assets/nodejs-project/bundle.cjs",
+    "prepare-backend-assets:windows": "mkdir .\\nodejs-assets\\nodejs-project || cd . && xcopy ..\\backend\\lib\\bundle.cjs .\\nodejs-assets\\nodejs-project\\ /Y",
```

Comment:
Is this change necessary, doesn't powershell understand both `mkdir`, `cd` and `cp`?

Reply:
`mkdir` differs in behaviour on Windows - it throws error and stops the process when directory already exists. That's why the `|| cd .` hack is here.
Also - yes, powershell has `cp` command but according to people's outputs in the task I saw that they were using regular cmd and cmd in windows does not have `cp`.
Of course we could add suggestion/requirement to the readme to use powershell but I decided that adjusting scripts to both cmd and powershell is not over the top since other commands (like mkdir) also work differently on Windows and Unix-like OSs.

Reply:
Ok, sounds good. Another alternative for the future could be having prepare scripts written in Javascript, so that we can maintain the same code for all platforms.